pic-sure-auth-services/src/main/java/edu/harvard/hms/dbmi/avillach/auth/filter/JWTFilter.java

```diff
@@ -183,9 +191,13 @@ private void setSecurityContextForUser(HttpServletRequest request, HttpServletRe
 
         logger.debug("User with ID: {} is active.", authenticatedUser.getUser().getUuid());
         logger.debug("Checking if user has accepted the latest terms of service.");
-        if (!tosService.hasUserAcceptedLatest(authenticatedUser.getUser().getSubject())) {
+
+        if (
+            !request.getRequestURI().endsWith("/tos/accept") && !tosService.hasUserAcceptedLatest(authenticatedUser.getUser().getSubject())
```

Comment:
Added `/tos/accept` exclusion so we can ensure that if they are on the accept route it doesn't matter if they have previously accepted terms or not.